packages/ra-ui-materialui/src/layout/AppBar.tsx

```diff
@@ -59,13 +59,16 @@ const useStyles = makeStyles(
 /**
  * The AppBar component renders a custom MuiAppBar.
  *
- * @param {ReactNode} children React node/s to be render as children of the AppBar
```

Comment:
This change isn't necessary

Reply:
The same (I think), you have only one param named `props` here and no description for it

Reply:
Done!